Intro tidyverse learnr (#8)
EDUCE-UBC/educer: inst/resources/markdown/how_to_follow_tutorials.Rmd

Throughout this tutorial, you will encounter text paragraphs such as this one, containing explanations and instructions. In text paragraphs, R code is always formatted with a monospace font and surrounded by a box, like `this`.  

You will also encounter coding exercises in interactive windows, such as the one pictured below. You can write and run code directly inside these interactive window ![](images/one.png){width=0.3in} and then click on the "Run Code" button to see the results ![](images/two.png){width=0.3in}. You can reset the exercise if you get stuck ![](images/three.png){width=0.3in}, or can get hints or the solution ![](images/four.png){width=0.3in}.

![](images/code_exercise_small.png){width=5in}

> In both text paragraphs and inside interactive code windows, whenever you see any code surrounded by `<angle_brackets>`, that always represents a placeholder, which you should replace with your own code.

Reviewer: ```suggestion
In both text paragraphs and inside interactive code windows, any placeholder you should replace with your own code is indicated by `<placeholder>`.
```
